machine/machine_core/machine.py

```diff
@@ -74,6 +75,9 @@ def __init__(self, address="127.0.0.1", image="unknown", verbose=False, label=No
         self.image = image
         self.atomic_image = self.image in ATOMIC_IMAGES
         self.ostree_image = self.image in OSTREE_IMAGES
+        self.dpkg_image = any(self.image.startswith(pfx) for pfx in DPKG_IMAGE_PREFIXES)
+        self.rpm_image = any(self.image.startswith(pfx) for pfx in RPM_IMAGE_PREFIXES)
```

Comment:
Would you consider a runtime detection instead? `image-customize` does that in `install_packages()`. Perhaps *that* bit would warrant a new public Machines API, `get_package_manager()`? Then we can extend this in the future for Alpine, zypper, etc, and the command would also work for our other images like "services".